src/c_ndp.hpp

```diff
@@ -100,7 +105,9 @@ const std::array<unsigned char, 94> c_ndp::generate_neighbor_advertisement_new(
 		m_generate_neighbor_advertisement_packet.at(56) = 0x00;
 		m_generate_neighbor_advertisement_packet.at(57) = 0x00;
 
-		uint16_t checksum = checksum_ipv6_packet(&m_generate_neighbor_advertisement_packet.front() + 22, &m_generate_neighbor_advertisement_packet.front() + 54, 40, 58);
+		tab_view<uint8_t> source_destination_addr(&m_generate_neighbor_advertisement_packet.front() + 22, 32);
+		tab_view<uint8_t> header_with_content(&m_generate_neighbor_advertisement_packet.front() + 54, 40);
```

Comment:
@robertoleksy @hbdgr @mikurys  FYI: we should aim to use this tab_view, but even if not, then at least instead
foo( arr.front() + 42 .... );
let's do
foo( & arr.at(42) ... ) or similar when possible, to have the assert that we are not outside array bounds